SelfReflection1-EmilyRowen.qmd

```diff
@@ -0,0 +1,61 @@
+
+    Preparing proposal
+
+**This Course**
+
+5.  Please tell me a little about your prior experience with R ("I use it all the time", "I am still learning", "I've never used it before"). What do you like most about working with R? What are the biggest obstacles for you when working with R?
+
+    I've taken one R course previously and used it to make visuals for poster presentations and my masters thesis. I haven't used it with spatial data before. I like the satisfaction of figuring out the puzzle of code. Recalling new functions or common code progressions is probably the biggest obstacle.
+
+6.  Describe your experience working with spatial data and GIS (either in R, or ArcGIS, or QGIS etc.). How do you tend to use spatial data (e.g., making maps, desiging field sampling, buidling stastical models)?
+
+    I've taken a course in ArcGIS and recently was challenged by Matt to try and make a map of Idaho to try and narrow my study area in QGIS. It went really well for never having used that software before! I definitely tend towards visualizations more than statistical processes.
+
+7.  How do you feel about taking this course (e.g., neutral, positive, negative) and why (e.g., aspects you are worried about, or looking forward to)?
+
+    Excited to get back into R and figure out how I might use it for my dissertation research!
+
+8.  After reviewing the syllabus, give some thought to your objectives for this course. Which parts of this course are going to be most useful to your research? What skills do you really want to focus on during the semester? Once you've given this some thought, craft 3-5 SMART objectives for you in this course. SMART objectives are **S**pecific (e.g., I want to make maps in R), **M**easurable (e.g., I want to build *3 maps displaying different elements of a location* in R), **A**chievable (e.g., I want to build 3 maps displaying different elements of a location *using ggplot*), **R**elevant (e.g., I want to build 3 maps displaying *social and ecological* elements of a location using ggplot), and **T**ime-bound (e.g., I want to build 3 maps displaying *social and ecological* elements of a location using ggplot *for the final project*). Try to think about your overall objectives for the course so that we can use to assess your final project. Remember, these objectives should be based in where you are now and where you'd like to be when the course ends. **If you're not sure how to answer this, schedule a meeting with me to chat about it**.
+
+    -   I would like to build at least 1 map that displays the status of water-ag conflicts in Idaho and 1 map that displays energy-ag conflict in Idaho.
+
+    -   I would like to perform some sort of statistical analysis on social and ecological elements of these conflicts (I won't know exactly which is appropriate until further into the course)
+
+    -   I would like to understand what visualizing qualitative-spatial data might mean and the challenges associated with it.
```

Comment:
Is there a way you can show that you've achieved this? Maybe a reflection in the final project or a specific type of data you will include?